### DISCUSSION TITLE
Add an option in the menu to mimic stable beat map sorting

### DISCUSSION THREAD

**Discussion by tylort69** [ppy/osu#17685] [Ideas] [answered]
a screenshot of stable sorting by rank achieved.
![screenshot008](https://user-images.githubusercontent.com/28652424/161983745-356d3093-90b1-4358-82b4-0f1661d5d947.png)

A screenshot of lazer sorting by rank achieved.
![osu_2022-04-06_09-51-08](https://user-images.githubusercontent.com/28652424/161990397-e191e8d0-ae80-42ca-b3c9-cec8cd83a221.png)

from what I can see it is not functioning in either of the ways that I would expect or at all according to [#4917](https://github.com/ppy/osu/issues/4917) ; 
1. The same behaviour seen in stable (global sorting regardless of parent mapset).
2. Sorting in each mapset locally.

"1" is the behaviour that I would like an option for if it is not intended to be implemented as default.

After further inspection it seems that the sort by Difficulty is not functioning as expected either, from what I can tell its sorting the mapsets (rather than individual maps) by the lowest legal difficulty contained in the mapset for your search parameters.

**Comment by bdach** [comment, marked as answer]
#2547
